content/en/_index.md

```diff
@@ -271,6 +311,17 @@ end
 // This is an example snippet for showing best practices.
 function example_snippet(string $projectId, string $filePath): void
 {{< /tab >}}
+{{< tab header="Terraform" >}}
+// [START product_example]
+#   Creates a VPC network
```

Comment:
Replace with `# This is an example snippet for showing best practices` to match others.

Would this be a good place to say whether we should include the `project` argument or not? I know some resources will just use the provider project, but others will explicitly look for the `project` argument.

If we do want to say something specifically about the project argument, we can mention that the `project` arg should use the `google_project` data object and include an example of that